webapp/channels/src/components/advanced_text_editor/use_key_handler.tsx

```diff
@@ -59,7 +58,6 @@ const useKeyHandler = (
     const lastChannelSwitchAt = useRef(0);
     const isNonFormattedPaste = useRef(false);
 
-    const latestReplyablePostId = useSelector((state: GlobalState) => (postId ? '' : getLatestReplyablePostId(state)));
```

Comment:
I love to see this sort of change since it means that we don't need to re-render this component or re-run the hook for something that doesn't affect visible rendering at all